script/lib/grd_string_replacements.py

```diff
@@ -22,9 +22,9 @@
 default_replacements = [
     (r'Brave Web Store', r'Web Store'),
     (r'Automatically send usage statistics and crash reports to Brave',
-     r'Automatically send crash reports to Brave'),
+     r'Automatically send reports to Brave when the browser crashes or hangs'),
     (r'Automatically sends usage statistics and crash reports to Brave',
-     r'Automatically sends crash reports to Brave'),
+     r'Automatically sends reports to Brave when the browser crashes or hangs'),
```

Comment:
In fact the diff in this file is only one primary change in the PR.
All the rest changes are generated by `npm run chromium_rebase_l10n`.
@Max @mariospr Is it the expected way to update such strings?

Reply:
I think this is fine as the strings are already in here and you are just tweaking them more.